CHANGELOG.md

```diff
@@ -2,6 +2,113 @@
+  - New method `taggable?` to programmatically determine if a Service Model class or instance is taggable.
+  - Generic Objects: Model updates
+    - Associations
+    - Tagging
+    - Service Methods: `add_to_service / remove_from_service`
+  - Git Automate support
+    - Branch/Tag support
+    - Contents are locked,and can be copied to other domains for editing
+    - Editable properties
+      - Enabled/Disabled
+      - Priority
+      - Removal of Domain
+    - Dedicated Server Role to store the repository
+- Platform
+  - PostgreSQL High Availability
+    - Primary/Standby DB config in Appliance Console
+    - Database-only appliance config in Appliance Console
+    - Failover Monitor
+  - Tenancy
+    - Groundwork in preparation for supporting multiple entitlements
+    - ApplicationHelper#role_allows and User#role_allows? combined and moved to Rbac
+    - Post refresh hook to queue mapping of Cloud Tenants
+  - Database maintenance scripts added to appliance
+- Providers
+  - Containers: Models for container deployments
+  - Google
+    - Preemptible Instances
+    - Retirement support
+  - Hawkular    
+    - Alerts
+       - Link miq alerts and hawkular events on the provider
+       - Convert miq alerts/profiles to hawkular group triggers/members of group triggers
+       - Sync the provider when miq alerts and alert profiles are created/updated
+   - Added entities: Domains and Server Groups
+       - Including their visualization in topology
+       - Limitation : still missing relevant icons that need to be designed by uxd
+   - Datasource entity now has deletion operation of the
+    - Support more event types for datasource and deployment
+    - Cross linking to VMs added to topology
+  - Microsoft Azure
+    - Additional Metrics
+    - Caching for SmartState
+  - OpenStack
+    - Host Aggregates
+    - Region Support
+  - Red Hat Enterprise Virtualization: Snapshot support
+  - VMware vSphere: Storage profiles
+- REST API
+  - Support for compressed ids in inbound requests
+  - CRUD support for Arbitration Rules
+- Service Broker
+  - Service Designer
+     - Blueprint API is 90% done, edit and publish are still in development
+     - Continue to connect the service designer with the API, create, view, and delete working
+  - Arbitration Profiles
+      - Collection of pre-defined settings
+      - Work in conjunction with the Arbitration Engine
+  - Rules Engine: API completed
+- SmartState
+  - Deployed new MiqDiskCache module for use with Microsoft Azure
+  - Scan time reduced from >20 minutes to <5 minutes
+  - Microsoft Azure backed read requests reduced from >7000 requests to <1000
+- User Interface
+  - I18n support for UI plugins
+  - Arbitration Profiles management for Service Broker
+  - Re-check Authentication button added to Provider list views
+  - Provisioning button added to the Templates & Images list and summary screens
+  - Subtype option added to Generic Catalog Items
+  - About modal added to OPS UI
```

Comment:
@chessbyte I think we should start standardizing on a term.  I think you and I were going with "Classic UI" or "Admin UI", but what's your opinion.